### PR TITLE
performance

### PR DESCRIPTION
**Pull request by PiDiBi**
parallel reading of files but serial processing because rest of code and sql scriptdom are not ready
was
https://github.com/tsqllint/tsqllint/pull/222

slightly worst but still great

Linted 1728 files in 4.5537925 seconds

### CODE REVIEW

**Comment by PiDiBi** [comment]
hey @nathan-boyd what do you think? my change doesn't change the code handling SQL validation (I discovered there is a problem within parser itself probably) but it just read files in paralell and then it runs old code one by one. it helps and it seems safe as far as old validating code is not changed

**Comment by nathan-boyd** [comment]
cool, taking a look

**Comment by PiDiBi** [comment]
hey, why is it still blocked ?

File: source/TSQLLint.Console/appsettings.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-    "debug":"false"
+    "debug":"true"
```
Comment (dougwilson) [inline source/TSQLLint.Console/appsettings.json]: Should this have been changed?

**Pull request merged**